src/app/public/modules/progress-indicator/progress-indicator-reset-button/progress-indicator-reset-button.component.html

```diff
@@ -1,7 +1,12 @@
-<button
-  class="sky-btn sky-btn-link"
-  type="button"
-  [disabled]="disabled"
-  (click)="resetProgress()">
-  <ng-content></ng-content>
-</button>
+<div
+  class="sky-progress-indicator-reset-button"
+>
```

Comment:
Nope; I prefer to wrap all components in a `div` with a CSS class name, but in this case we can remove it.

Reply:
Is this div necessary? We don't have a wrapper div around the other nav buttons.